src/dbnode/generated/proto/namespace/schema.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package namespace;
+
+// SchemaOptions contains schema information for a namespace.
+message SchemaOptions {
+    // history contains a history of deployed schema definitions.
+    SchemaHistory history = 1;
+    // defaultMessageName identifies the proto message that contains the default schema for the namespace.
+    string defaultMessageName = 2;
+}
+
+// SchemaHistory is versioned FileDescriptorSet.
+message SchemaHistory {
+    // versions is a list of FileDescriptorSet sorted by version in ascending order.
+    repeated FileDescriptorSet versions = 1;
+}
+
+// FileDescriptorSet is a set of proto file descriptors.
+message FileDescriptorSet {
+    // id identifies a deployed version of FileDescriptorSet.
+    string deployId = 1;
+    // prevId identifies the previous deploy id of FileDescriptorSet.
```

Comment:
super nit: Want to just leave a comment saying the previous one can be inferred from the ordering but we add this as a precaution?

Reply:
will do